app/services/coronavirus_pages/timeline_entry_builder.rb

```diff
@@ -0,0 +1,27 @@
+class CoronavirusPages::TimelineEntryBuilder
+  def create_timeline_entries
+    if timeline_entries_from_yaml.any?
+      Announcement.transaction do
```

Comment:
Copy and paste error: This transaction should be on `TimelineEntry`.

Reply:
sorted :)